content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -119,41 +77,39 @@ the subtotals between Seller Central and Odoo.
 marketplaces.
 
 .. tip::
    To find out if a marketplace is eventually compatible, check the `Amazon Documentation
-   <https://docs.developer.amazonservices.com/en_US/dev_guide/DG_Endpoints.html>`_. The marketplace
+   <https://developer-docs.amazon.com/amazon-shipping/docs/seller-central-urls>`_. The marketplace
    must belong to the same region as that of your seller account.
 
-To add a new marketplace, you must first activate the :ref:`developer mode <developer-mode>`.
+To add a new marketplace, activate the :ref:`developer mode <developer-mode>`.
 
 Once that is done, go to :menuselection:`Sales --> Configuration --> Settings --> Connectors -->
 Amazon Sync --> Amazon Marketplaces`.
 
-From there, you can create a new marketplace record. You will need the Marketplace ID and Endpoint
-for your marketplace as described in the `Amazon Documentation
-<https://docs.developer.amazonservices.com/en_US/dev_guide/DG_Endpoints.html>`_.
+From there, create a new marketplace record. Enter the Marketplace ID and select the Amazon Region
+for your marketplace as described in the `Amazon Documentation for marketplace IDs
+<https://developer-docs.amazon.com/amazon-shipping/docs/marketplace-ids>`_, and the Seller Central
+URL as described in the `Amazon Documentation for seller central URLs
+<https://developer-docs.amazon.com/amazon-shipping/docs/seller-central-urls>`_.
 
-Set the name of the record to ``Amazon.<domain>`` to easily retrieve it (for instance:
-``Amazon.se``). The **Code**, **Domain** and **API Identifier** fields should respectively hold
-the *Country Code*, *Amazon MWS Endpoint* and *MarketplaceId* values from the Amazon Documentation.
+Set the name of the record to ``Amazon.<country code>`` to easily retrieve it (e.g.: ``Amazon.se``).
+The :guilabel:`API Identifier`, the :guilabel:`Region` and the :guilabel:`Seller Central URL` fields
+should respectively hold the *MarketplaceId*, the selected Amazon region and the *Seller Central
+URL* values from the Amazon Documentation.
 
-Once the marketplace is saved, you should then update the Amazon Account configuration by going to 
+Once the marketplace is saved, update the Amazon Account configuration by going to
 :menuselection:`Sales --> Configuration --> Settings --> Connectors --> Amazon Sync -->
-Amazon Accounts`, open the account on which you wish to use the new marketplace, go to the
-**Marketplaces** tab and click on **Update available marketplaces** (an animation should confirm the
-success of the operation). Newly added marketplaces are automatically added to the list of
-synchronized marketplaces. If the new marketplace is not added to the list, it means that it is
-either incompatible or unavailable for your seller account.
-
-.. important::
-   As manually added marketplaces are not officially supported by Odoo, there is no guarantee that
-   adding one as described above will work, nor can this be considered as a bug when contacting Odoo
-   Support.
```

Comment:
Same here.

Reply:
I'm not sure... :thinking: This should work whatever the marketplace is, the SP-API is more consistent for its variable names than MWS was

Reply:
After discussing it, this'll be changed in master